pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.springframework</groupId>
```

Comment:
These three tags are used by maven to create and name your jar when you make a build.

https://maven.apache.org/guides/mini/guide-naming-conventions.html

You want your groupId to match with your package name in src/main/java. So you could do something like org.galatea.jingyang as your group id and change your package structure to src/main/java/org/galatea/jingyang